galeracluster/source/index.rst

```diff
@@ -53,14 +53,24 @@ The following features are available through Galera Cluster:
 
 The result is a high-availability solution that is both robust in terms of data integrity and high-performance with instant failovers.
 
+
```

Comment:
This block of text now appears on the title page of the documentation, further pushing down the table of contents way beyond the bottom of the screen.

I think "what is new" needs to be a separate page. Also, the new functions that were introduced deserve a mention in the "What is new" page.